doc/examples/transform/plot_geometric.py

```diff
@@ -112,5 +113,10 @@
     a.axis('off')
 
 plt.tight_layout()
-
 plt.show()
+
+######################################################################
+# For an estimation robust to outliers using the 
+# `RANSAC algorithm <https://en.wikipedia.org/wiki/Random_sample_consensus>`_, 
+# see the :ref:`sphx_glr_auto_examples_transform_plot_matching.py` tutorial.
```

Comment:
I'm not sure to see the point of this remark at this stage. I think that this sentence will loose the newcomer. I would remove it and instead point toward the text in the user guide that you wrote and which explains it very well.